src/main/kotlin/no/nav/arbeidsgiver/altinnrettigheter/proxy/tilgangskontroll/TilgangskontrollService.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.arbeidsgiver.altinnrettigheter.proxy.tilgangskontroll
+
+import org.springframework.stereotype.Service
+
+@Service
+class TilgangskontrollService(val tokenUtils: TilgangskontrollUtils) {
+
+    fun hentInnloggetBruker(): InnloggetBruker {
+        return if (tokenUtils.erInnloggetSelvbetjeningBruker()) {
+            tokenUtils.hentInnloggetSelvbetjeningBruker()!!
```

Comment:
Hva betyr !! her?

Reply:
Den skal kaste en `KotlinNullPointerException` hvis `InnloggetBruker` er null. Men jeg tror ikke det er riktig sted for å håndtere det. Vil helst sikre det i `hentInnloggetSelvbetjeningBruker()` og kaste en `TilgangskontrollException` dersom ingen `InnloggetBruker` er funnet.